projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn-base.json

```
@@ -0,0 +1,211 @@
+    "item.refurbished_furniture.cooked_vegetable_pizza": "熟蔬菜披萨",
+    "item.refurbished_furniture.vegetable_pizza_slice": "蔬菜披萨片",
+    "item.refurbished_furniture.raw_meatlovers_pizza": "生肉披萨",
+    "item.refurbished_furniture.cooked_meatlovers_pizza": "熟肉披萨",
+    "item.refurbished_furniture.meatlovers_pizza_slice": "肉披萨片",
+    "container.refurbished_furniture.workbench": "工作台",
+    "container.refurbished_furniture.drawer": "抽屉",
+    "container.refurbished_furniture.crate": "板条箱",
+    "container.refurbished_furniture.kitchen_drawer": "厨房抽屉",
+    "container.refurbished_furniture.cooler": "冷藏箱",
+    "container.refurbished_furniture.fridge": "冰箱",
+    "container.refurbished_furniture.freezer": "冰柜",
+    "container.refurbished_furniture.microwave": "微波炉",
+    "container.refurbished_furniture.stove": "炉灶",
+    "container.refurbished_furniture.mailbox": "邮箱",
+    "container.refurbished_furniture.post_box": "邮筒",
+    "container.refurbished_furniture.electricity_generator": "发电机",
+    "container.refurbished_furniture.recycle_bin": "回收桶",
+    "container.refurbished_furniture.storage_cabinet": "储物柜",
+    "gui.refurbished_furniture.set_mailbox_name": "设置邮箱名称",
+    "gui.refurbished_furniture.rename_mailbox_failed": "更新邮箱名称失败",
+    "gui.refurbished_furniture.mailboxes": "邮箱",
+    "gui.refurbished_furniture.search": "搜索……",
+    "gui.refurbished_furniture.search_mailboxes": "搜索邮箱",
+    "gui.refurbished_furniture.enter_message": "输入消息……",
+    "gui.refurbished_furniture.package_message": "包裹消息",
+    "gui.refurbished_furniture.send": "发送",
+    "gui.refurbished_furniture.how_to": "如何操作",
+    "gui.refurbished_furniture.post_box_info": "从列表中选择一个邮箱，你可以通过名称搜索邮箱，或者通过在名称前加上@来搜索玩家，将物品放入包裹槽中，并可选择包含一条消息，点击发送按钮以递送包裹。",
+    "gui.refurbished_furniture.workbench_info": "从列表中选择一个合成配方，并收集所需的材料来制作物品，工作台会搜索你的背包、自身存储以及相邻的存储方块中的材料。",
+    "gui.refurbished_furniture.package_sent_by": "由%s发送",
+    "gui.refurbished_furniture.package_open": "右击打开",
+    "gui.refurbished_furniture.set_doorbell_name": "设置门铃名称",
+    "gui.refurbished_furniture.doorbell_rang": "门铃响了",
+    "gui.refurbished_furniture.status.online": "在线",
+    "gui.refurbished_furniture.status.offline": "离线",
+    "gui.refurbished_furniture.status.overloaded": "过载",
+    "gui.refurbished_furniture.status.no_fuel": "燃料耗尽",
+    "gui.refurbished_furniture.node_count": "%s / %s",
+    "gui.refurbished_furniture.recycle": "回收",
+    "gui.refurbished_furniture.save": "保存",
+    "gui.refurbished_furniture.next_preset": "下一个预设",
+    "gui.refurbished_furniture.previous_preset": "上一个预设",
+    "gui.refurbished_furniture.no_power": "缺少电力",
+    "gui.refurbished_furniture.link_too_long": "距离过远",
+    "gui.refurbished_furniture.link_too_many": "链接过多",
+    "gui.refurbished_furniture.link_already_connected": "已连接",
+    "gui.refurbished_furniture.link_invalid_node": "无效节点",
+    "gui.refurbished_furniture.link_unpowerable": "链路无法供电",
+    "gui.refurbished_furniture.link_outside_area": "链路超出供电区域",

```

```suggestion
    "gui.refurbished_furniture.link_unpowerable": "链接无法供电",
    "gui.refurbished_furniture.link_outside_area": "链接超出供电区域",
```
建议（可不更改但推荐更改）
与上方L101统一名称，链路这个名称比较偏通信吧，如改则全文通查